package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@collaborne/material-kanban",
-  "version": "2.0.3",
+  "version": "2.1.0",
```

Comment:
This is weird, I wouldn't expect a PR to bring in a version bump (and therefore some tags must exist on a "wip" branch?)

Reply:
Hmm, this is indeed not ideal. Tags shouldn't be created from branches.

Yet, the tag already exists now (and it's on npmjs: https://www.npmjs.com/package/@collaborne/material-kanban).
So, I'd propose to keep the tag and merge the PR like this to `main` (to have code consistent with tags and preserve history).

@ankon Or you think we should rather remove the tag (and thereby alter history)?

Reply:
Altering history might be more ugly indeed. 

So I guess merging this (after reviews, and without rebasing!), and then doing a cleanup 2.2.0 or such would work.